Ullaakut/cameradar
# Fix attack loop and add reolink routes

## Goal of this PR

Fixes bug #213 introduced in #209 

## How to test it

* Start an RTSP server with authentication and a route, using `gst-rtsp-server` or a camera model which doesn't follow the RTSP RFC
* Run cameradar on it
* Logs should show that it runs the route attack twice and the stream validation twice
* Cameras should be accessed successfully
